Text/Mustache/Compile.hs

```diff
@@ -15,6 +15,7 @@
 
 module Text.Mustache.Compile
   ( compileMustacheDir
+  , compileMustacheDirCustom
```

Comment:
You also need to export `compileMustacheDirCustom` from `Text.Mustache`.

Reply:
Fixed.